Earthfile

```diff
@@ -0,0 +1,35 @@
+VERSION 0.8
+
+
+INSTALL_PACKAGE: 
+  FUNCTION
+  ARG package
+  COPY +${package}/${package}*.deb ./
+  RUN dpkg -i *.deb
+
+dpkgbuild:
+  FROM debian:bookworm
+  RUN apt-get update
+  RUN apt-get install -y debhelper lintian
+
+ros-archive-keyring:
+  FROM +dpkgbuild
+  RUN mkdir /tmp/pkg
+  WORKDIR /tmp/pkg
+  COPY README .
+  COPY debian debian
+  COPY keys keys
+  RUN dpkg-buildpackage
+  RUN lintian 
+  SAVE ARTIFACT ../*.deb AS LOCAL output/
+
+
+test-keyring-pkg-install:
+  # Test that keyring package can be installed and configures the keys appropiately
+  FROM ubuntu:noble
```

Comment:
A future PR could make this a parameter to allow overriding the test environment to e.g. test on Debian Trixie or Ubuntu Jammy.